bin/minifollowups/pycbc_sngl_minifollowup

```diff
@@ -124,9 +126,26 @@ insp_data_seglists.coalesce()
 num_events = int(workflow.cp.get_opt_tags('workflow-sngl_minifollowups',
                  'num-sngl-events', ''))
 
+# This helps speed up the processing to ignore a large fraction of triggers
+snr_mask = None
+if args.min_snr:
+    logging.info('Calculating Prefilter')
+    f = hdf.HFile(args.single_detector_file, 'r')
+    idx, _ = f.select(lambda snr: snr > args.min_snr,
+                      '{}/snr'.format(args.instrument),
+                      return_indices=True)
+    snr_mask = numpy.zeros(len(f['{}/snr'.format(args.instrument)]),
+                           dtype=bool)
+    snr_mask[idx] = True
+
 trigs = hdf.SingleDetTriggers(args.single_detector_file, args.bank_file,
                               args.veto_file, args.veto_segment_name,
-                              None, args.instrument)
+                              None, args.instrument, premask=snr_mask)
+
+def restrict_triggers_to_logic_mask(triggers, logic_mask):
```

Comment:
This also looks like its better as a method of 'SingleDetTriggers` (if such a method isn't already there).

Reply:
Ok, done.